[tftrt update]

jjsjann123 opened the pull request:
Added python tests for converter functions
  Added BUILD for python tests

jjsjann123 committed:
[tftrt update]
  Added python tests for converter functions
  Added BUILD for python tests

jjsjann123 commented:
@aaroey Here's the unit testing scripts. I was using yapf with {google style + indent_width: 2}
Also a reminder that this scripts depends on functionalities provided from PR #20350

aaroey commented:
Thanks @jjsjann123, as mentioned in #20350 I'll merge this with tf_trt_integration_test.py

aaroey committed:
Refactor tf_trt_integration_test so we can extend it to other graphs.

aaroey committed:
Refactor the test base class so each subclass can define its own graph.

aaroey committed:
Fix conflicts with upstream.

aaroey committed:
Add support for multiple inputs.

aaroey committed:
Fix conflicts with upstream

jjsjann123 committed:
[tftrt unit tests]
  1. unit tests refactored to tf_trt_integration_test style
  2. disabled failed test (tests passed in static conversion but failed in
     dynamic conversion)
  3. comment on _VerifyGraphDef:
     we should cover cases for failed conversion in dynamic conversion,
     where func def (TF fallback) is used.

aaroey committed:
Fix python formatting issues.

aaroey committed:
Fix dependency issues.

aaroey committed:
Disable unused import errors.
